src/foam/nanos/notification/email/EmailTemplate.js

```diff
@@ -62,7 +62,8 @@ bodyAsByteArrayIsSet_ = false;`
     {
       class: 'String',
       name: 'sendTo',
-      documentation: `This property will set who the email is being sent too.`
+      documentation: `This property will set to whomever the email is being sent
+        to.`
```

Comment:
Isn't this injecting linefeed into the string and splitting it over 2 lines?

Reply:
yes.  There are foam tools to build documentation from the documentation properties, not sure how things are rendered.

Reply:
I guess if its rendered as HTML the LF will be ignored.

Reply:
I split it over two lines to maintain the code style. The line feed is ignored during rendering.